src/transform.jl

```diff
@@ -29,11 +29,11 @@ end
 scale(tf::AffineTransform, s::Vector) = AffineTransform(scale(tf.scalefwd, s), tf.offset)
 scale(s::Vector, tf::AffineTransform) = AffineTransform(scale(s, tf.scalefwd), tf.offset.*s)
 
-*(a::AffineTransform, v::AbstractVector) = tformfwd(a, v)
-\(a::AffineTransform, x::AbstractVector) = tforminv(a, x)
+*(a::AffineTransform, v::AbstractVecOrMat) = tformfwd(a, v)
+\(a::AffineTransform, x::AbstractVecOrMat) = tforminv(a, x)
 
-tformfwd{T}(a::AffineTransform{T}, x::AbstractVector) = tformfwd!(Array(typeof(one(T)*one(eltype(x))), length(x)), a, x)
-tforminv{T}(a::AffineTransform{T}, x::AbstractVector) = tforminv!(Array(typeof(one(T)*one(eltype(x))), length(x)), a, x)
+tformfwd{T}(a::AffineTransform{T}, x::AbstractVecOrMat) = tformfwd!(Array(typeof(one(T)*one(eltype(x))), length(x)), a, x)
+tforminv{T}(a::AffineTransform{T}, x::AbstractVecOrMat) = tforminv!(Array(typeof(one(T)*one(eltype(x))), length(x)), a, x)
```

Comment:
The change here doesn't seem like it will do much. If it's an `AbstractMatrix` (which your `a[1,:]'` is), the next two lines will have priority.